src/pages/Signup/Singup.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Navbar from '../../components/Navbar/Navbar';
+import { useState } from 'react';
+import '../../pages/Login/Login.css';
+function Singup() {
+  const [user, setUser] = useState({
+    name: '',
+    email: '',
+    password: '',
+    acceptCondition: false,
+  });
+  const changeHandler = e => {
+    setUser(prevUser => {
+      return {
+        ...prevUser,
+        [e.target.name]: e.target.value,
+      };
+    });
+  };
+  console.log(user);
+  return (
+    <>
+      <Navbar />
+      <div className="login-container">
+        <div className="login-card">
+          <h2>Sign Up</h2>
+          <form>
+            <label htmlFor="name"> Name </label>
+            <input
+              id="name"
+              name="name"
+              value={user.name}
+              onChange={changeHandler}
+              className="form-input"
+              type="text"
+            />
+            <label htmlFor="email"> Email Address </label>
+            <input
+              id="email"
+              name="email"
+              value={user.email}
+              onChange={changeHandler}
+              className="form-input"
+              type="text"
```

Comment:
since this input will be used for email, I would suggest to use the `type='email'` attribute